src/theme.ts

```diff
@@ -8,5 +8,46 @@ export const theme: DefaultTheme = {
   colors: {
     primary: '#007896',
     secondary: '#163150'
+  },
+  fontSizes: ['10px', '12px', '16px', '18px', '20px', '40px'],
+  lineHeights: ['16px', '24px', '48px'],
+  text: {
+    heading: {
+      fontFamily: 'body',
+      fontSize: 5,
+      lineHeight: 3,
+      fontWeight: 700,
+      color: 'DARK_BLUE'
```

Comment:
Prefer theme named colors.

Reply:
I would like to rename the colors, but I don't know how we would convert the current colors to a theme color scheme. We might need design to come up with this?